backend/core/scripts/import-listing-from-json-file.ts

```diff
@@ -1,4 +1,4 @@
-import { importListing } from "./listings-importer"
+import { importListing } from "./import-helpers"
```

Comment:
This may be a noob question, but why would this script fail but the csv script work?

Reply:
I added the code you suggested and that seemed to fix it, but now I'm getting a foreign key error. Trying to determine if it's caused by the changes or if it existed before.

Reply:
Can you paste the foreign key error you're seeing? (Or ping and we can figure out over chat?)

Reply:
> This may be a noob question, but why would this script fail but the csv script work?

`serviceOptions` seems to be some kind of global variable (that is defined/instantiated in backend-swagger.ts), and it's `axios` variable controls the details of the connection with the backend. So at some point in the script, we need to call `serviceOptions.axios = <something with backend connection details>`. The CSV script does that in `getJurisdictions` (before it calls `importListing`), but the JSON script doesn't call `getJurisdictions`, so it hasn't ever set `serviceOptions.axios` by the time it calls `importListing`.

Reply:
Got it, thanks for clarifying! As discussed offline, FK error was because jurisdiction wasn't added to the json script. I added it and the script passes for me now.